app/parse.py

```diff
@@ -8,8 +15,45 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_single_qoute(qoute_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=qoute_soup.select_one(".text").text,
+        author=qoute_soup.select_one(".author").text,
+        tags=[tag.text for tag in qoute_soup.select(".tag")],
+    )
+
+
+def get_list_qoute() -> [Quote]:
+    value = []
+    num_page = 1
+    while num_page:
+        url_num_page = urljoin(BASE_URL, f"page/{num_page}")
+        page = requests.get(url_num_page).content
+        soup = BeautifulSoup(page, "html.parser")
+        qoutes = soup.select(".quote")
+        if qoutes:
+            print(f"Start page #{num_page}")
+            value.extend(
+                [parse_single_qoute(qoute_soup) for qoute_soup in qoutes]
+            )
+            num_page += 1
+        else:
+            return value
+
+
+def write_qoutes_to_csv(qoutes: [Quote], output_csv_path: str) -> None:
```

Comment:
invalid "quotes" name
quotes, not qoutes
fix it everywhere
+
invalid typehint
use list[Quote]